package.json

```diff
@@ -30,6 +31,7 @@
     "clsx": "^2.1.1",
     "date-fns": "^3.6.0",
     "date-fns-tz": "^3.1.3",
+    "ethers": "^6.13.2",
```

Comment:
Do not introduce redundant dependencies, this project already use `viem` and you must stick to it (see https://viem.sh/docs/ethers-migration).